src/main/groovy/life/qbic/datamodel/dtos/business/facilities/Facility.groovy

```diff
@@ -0,0 +1,47 @@
+package life.qbic.datamodel.dtos.business.facilities
+
+/**
+ * <p>Controlled vocabulary for concise core facility labels.</p>
+ *
+ * <p>The following facilities are currently defined:</p>
+ *
+ * <ul>
+ *     <li>CFMB: Core Facility for Medical Bioanalytics</li>
+ *     <li>IMGAG: Institute for Medical Genetics and Applied Genomics</li>
+ *     <li>MGM: Institute for Medical Microbiology and Hygiene</li>
```

Comment:
Why is it `MGM` what does that stand for? Why not `MMH`or something similar? Isn't it https://uni-tuebingen.de/en/faculties/faculty-of-science/departments/interfaculty-facilities/imit/research-groups/medical-microbiology-and-hygiene/

Reply:
Thats the official acronym, see the offer footer please